docs/project/2024/s_m.takdeer/index.md

```diff
@@ -0,0 +1,85 @@
+
+## Motivation
+I chose this project to delve into the world of wearable technology and sensor integration, driven by an interest in health and fitness. The challenge of creating a device that assists in health management and fitness tracking using real-time data was highly appealing.
+
+## Architecture
+The system architecture consists of several key components:
+
+Microcontroller: Serves as the central processing unit, managing sensor data and wireless communication.
+Sensors: Includes an accelerometer for step tracking, a heart rate sensor for monitoring pulse, and a GPS module for location tracking.
+Display/Indicators: A small display or LED indicators provide visual feedback and data display directly on the device.
+Communication Module: Handles data transmission between the fitness tracker and a smartphone.
+These components are interconnected, with the microcontroller at the core processing inputs from the sensors and sending information to the display and through the communication module.
+
+## Log
+
+Week 6 - 12 May: 
+Week 7 - 19 May: 
+Week 20 - 26 May: 
+
+
+## Hardware
+
+    Microcontroller: Raspberry Pi Pico W
+Functionality: Acts as the central processing unit of the fitness tracker. It handles data collection from sensors, processes this data, and manages wireless communication.
+Reason for Selection: The Raspberry Pi Pico W is selected for its compact size, low power consumption, integrated Wi-Fi for data transmission, and adequate processing power to handle multiple sensor data streams.
+
+    Accelerometer
+Functionality: Measures acceleration forces that can be used to calculate steps and movement. It detects changes in motion and orientation.
+Reason for Selection: Essential for a fitness tracker as it enables the device to track steps accurately, which is a fundamental feature for activity monitoring.
+
+    Heart Rate Sensor
+Functionality: Monitors the heart rate of the user by detecting the pulse via optical or electrical methods.
+Reason for Selection: Heart rate monitoring is crucial for tracking fitness and health metrics, particularly in exercises involving cardiovascular activity.
+
+    GPS Module
+Functionality: Provides precise location tracking capabilities, which allows the fitness tracker to record routes and distances traveled.
+Reason for Selection: GPS functionality enhances the tracker by enabling outdoor sports enthusiasts to map their runs, hikes, or bike rides.
+
+    Small Display or LED Indicators
+Functionality: Displays real-time data such as step count, heart rate, and GPS maps. LED indicators can provide simple feedback like heart rate zones or connectivity status.
+Reason for Selection: A display enriches the user interface by providing a visual readout of data, while LED indicators offer minimalistic, power-saving feedback.
+
+## Schematics
+Place your KiCAD schematics here (not provided in text format).
+
+## Bill of Materials
+
+Device	                |  Usage	             |  Price
```

Comment:
Please format the table correctly and verify that it displays using `npm start`.